### PR TITLE
keyspace: prohibit merging the default keyspace group

### PR DESCRIPTION
**Pull request by JmPotato**
### What problem does this PR solve?

Issue Number: ref #6589.

### What is changed and how does it work?

```commit-message
Prohibit merging the default keyspace group.
```

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by binshi-bing — APPROVED** [review]

**Review by rleungx — APPROVED** [review]

File: pkg/keyspace/tso_keyspace_group.go [hunk 1]
```diff
@@ -762,6 +762,9 @@ func (m *GroupManager) IsExistNode(addr string) bool {
 
 // MergeKeyspaceGroups merges the keyspace group in the list into the target keyspace group.
 func (m *GroupManager) MergeKeyspaceGroups(mergeTargetID uint32, mergeList []uint32) error {
+	if mergeTargetID == utils.DefaultKeyspaceGroupID {
```
Comment (lhy1024) [inline pkg/keyspace/tso_keyspace_group.go:765]: mergeTargetID? mergeList?

**Pull request merged**